templates/registry/markdown/entity_namespace.md.j2

```diff
@@ -0,0 +1,46 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_namespace.md.j2 -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+{%- import 'stability.j2' as stability %}
+{%- import 'attribute_table.j2' as at %}
+
+# {{ ctx.id | title_case }}
+
+{% for e in ctx.groups | sort(attribute='name') %}
+{%- set id_attrs = e.attributes | selectattr("role", "equalto", "identifying") -%}
+{%- set desc_attrs = e.attributes | selectattr("role", "equalto", "descriptive") -%}
+{%- set misc_attrs = e.attributes | rejectattr("role", "defined") -%}
+
+## {{ e.name | title_case }}
+
+**Status:** {{ stability.badge(e.stability, e.deprecated) }}
+
+**type:** `{{ e.name }}`
+
+**Description:** {{ e.brief | trim }}
+
+{%- if desc_attrs | length > 0 %}
+
+**Identifying Attributes:**
+
+{{ at.generate(id_attrs, "", "/docs/registry/attributes", e.lineage.attributes) | trim }}
+{%- endif %}
+{%- if desc_attrs | length > 0 %}
+
+**Descriptive Attributes:**
+
+{{ at.generate(desc_attrs, "", "/docs/registry/attributes", e.lineage.attributes) | trim }}
+{%- endif %}
+{%- if misc_attrs | length > 0 %}
+
+**Other Attributes:**
+
+_Note: Stable Entities MUST have attributes with a defined role._
```

Comment:
```suggestion
> :warning: This entity definition contains attributes without a role. Stable Entities MUST NOT have attributes without a defined role.
```

so it gives a warning to conventions authors that something needs to be changed in yaml and also tells how to fix it?

Reply:
Fixed!, good suggestion.